ui/test/services/bom_bar_client_test.dart

```diff
@@ -160,6 +160,34 @@ void main() {
         });
       });
 
+      group('Find obligations for dependencies of a project', () {
+        test('finds obligations per dependency', () async {
+          server.respondJson({
+            'obligation': [
```

Comment:
Suggestion to start using constants for "'obligation'" and "`DependencyId`" to make the test more concise. (Note that you can put such constants in the scope of the group.)